src/SprykerEco/Zed/ComputopApi/ComputopApiDependencyProvider.php

```diff
@@ -25,13 +25,13 @@ public function provideBusinessLayerDependencies(Container $container): Containe
     {
         $container = parent::provideBusinessLayerDependencies($container);
 
-        $container[static::SERVICE_COMPUTOP_API] = function () use ($container) {
+        $container->set(static::SERVICE_COMPUTOP_API, function () use ($container) {
```

Comment:
* Extract each dependency to the separate method as we usually do for regular modules.
* `addComputopApiService()`.
* `addStoreFacade()`.

Reply:
Please, make this changes.